include/revng/Support/TarIO.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/Support/MemoryBuffer.h"
+
+#include "revng/ADT/CUniquePtr.h"
+#include "revng/Support/Generator.h"
+
+#include "archive.h"
```

Comment:
Is this a system header? We need to put it in the right spot!

Reply:
We need to add it to `.clang-format`.

Reply:
Is this enough or are you thinking of a more general rule?

```
      { Regex: '^"archive\.h"$', Priority: -8 },
```